tox.ini

```diff
@@ -17,6 +17,7 @@ deps=
     pytest-cov
     hypothesis[numpy] !=3.5.1, !=3.5.0
     numpy
+    pandas
```

Comment:
It'd be nice to have an environment without Pandas; what about moving pandas dependency to testenv, so that -nodeps environments don't have Pandas?